lib/route.js

```diff
@@ -85,6 +86,10 @@ function buildRouting (options) {
 
       globalExposeHeadRoutes = options.exposeHeadRoutes
       disableRequestLogging = options.disableRequestLogging
+      if (typeof disableRequestLogging === 'function') {
```

Comment:
check is placed here so that we don't need to repeat it on every request, which is a hot path